### ISSUE TITLE
thread pool metrics duplication

### ISSUE THREAD

**Issue by FrankChen021** [FrankChen021/bithon#164]
```
insert into "bithon_thread_pool_metrics" ("timestamp", "appName", "instanceName", "executorClass", "poolName", "callerRunTaskCount", "abortedTaskCount", "discardedTaskCount", "discardedOldestTaskCount", "exceptionTaskCount", "successfulTaskCount", "totalTaskCount", "activeThreads", "currentPoolSize", "maxPoolSize", "largestPoolSize", "queuedTaskCount") values (timestamp '2021-09-30 19:23:38.899', 'clickhouse-gateway-dev', '192.168.2.241:9090', 'reactor.core.scheduler.ReactorThreadFactory', '639690519', 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0) [23505-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:459) ~[h2-1.4.200.jar:1.4.200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429) ~[h2-1.4.200.jar:1.4.200]
	at org.h2.message.DbException.get(DbException.java:205) ~[h2-1.4.200.jar:1.4.200]
	at org.h2.message.DbException.get(DbException.java:181) ~[h2-1.4.200.jar:1.4.200]
	at org.h2.index.BaseIndex.getDuplicateKeyException(BaseIndex.java:103) ~[h2-1.4.200.jar:1.4.200]
	at org.h2.mvstore.db.MVSecondaryIndex.checkUnique(MVSecondaryIndex.java:221) ~[h2-1.4.200.jar:1.4.200]
	at org.h2.mvstore.db.MVSecondaryIndex.add(MVSecondaryIndex.java:196) ~[h2-1.4.200.jar:1.4.200]
	at org.h2.mvstore.db.MVTable.addRow(MVTable.java:531) ~[h2-1.4.200.jar:1.4.200]
	at org.h2.command.dml.Insert.insertRows(Insert.java:195) ~[h2-1.4.200.jar:1.4.200]
	at org.h2.command.dml.Insert.update(Insert.java:151) ~[h2-1.4.200.jar:1.4.200]
	at org.h2.command.CommandContainer.update(CommandContainer.java:198) ~[h2-1.4.200.jar:1.4.200]
	at org.h2.command.Command.executeUpdate(Command.java:251) ~[h2-1.4.200.jar:1.4.200]
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:168) ~[h2-1.4.200.jar:1.4.200]
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:768) ~[h2-1.4.200.jar:1.4.200]
	at com.alibaba.druid.pool.DruidPooledStatement.executeBatch$original$84weqDUU(DruidPooledStatement.java:791) ~[druid-1.1.16.jar:1.1.16]
	at com.alibaba.druid.pool.DruidPooledStatement.executeBatch$original$84weqDUU$accessor$UAWUnSFq(DruidPooledStatement.java) ~[druid-1.1.16.jar:1.1.16]
	at com.alibaba.druid.pool.DruidPooledStatement$auxiliary$xKdbphsw.invoke(Unknown Source) ~[druid-1.1.16.jar:1.1.16]
	at com.sbss.bithon.agent.bootstrap.aop.AroundMethodAop.intercept(AroundMethodAop.java:64) ~[na:na]
	at com.sbss.bithon.agent.bootstrap.aop.MethodAop.intercept(MethodAop.java:47) ~[na:na]
	at com.alibaba.druid.pool.DruidPooledStatement.executeBatch(DruidPooledStatement.java) ~[druid-1.1.16.jar:1.1.16]
	at org.jooq.tools.jdbc.DefaultStatement.executeBatch(DefaultStatement.java:112) ~[jooq-3.13.6.jar:na]
	at org.jooq.tools.jdbc.DefaultStatement.executeBatch(DefaultStatement.java:112) ~[jooq-3.13.6.jar:na]
	at org.jooq.impl.BatchMultiple.execute(BatchMultiple.java:113) [jooq-3.13.6.jar:na]
	... 10 common frames omitted
```

**Comment by FrankChen021** [comment]
Should aggregate at the agent side because poolId is meaningless at server side or user side. We can add a new field such as poolCount to know how many instances of a pool with same class

**Comment by FrankChen021** [comment]
Closed via #520
